src/Targets/Microsoft.Managed.DesignTime.targets

```diff
@@ -41,11 +41,6 @@
       References capability, but everything else we want back.
     -->
     <ProjectCapability Include="
-                          AssemblyReferences;
```

Comment:
At some point the project system will be used for more than .Net core and each project type (what used to be flavors) need control over their own behavior. For example, UWP projects should display the SDK references tab while a desktop project will want the COM reference tab. Those capabilities need to come from what controls those specific types (which I'm presuming are going to be either different SDK's or different targets within the SDK).

That being said, Lifeng proposed some extensibility points that will allow us more freedom over how to control these instead of these capabilities so if those work we won't need either of these and would be defining specific project type capabilities that we can use on the project side to control things.